api/middleware/validate-middleware.js

```diff
@@ -1,3 +1,5 @@
+const Tabs = require('../../models/tabs-model.js')
```

Comment:
Notice in some files you have " but others you have ', you may want to setup a format on save with VS Code